website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

```diff
@@ -89,7 +89,54 @@ Select **Allow**. This redirects you back to <Constant name="cloud" />. You are
 Workload Identity Federation (WIF) allows application workloads, running externally to <Constant name="cloud" />, to act as a service account without the need to manage service accounts or other keys for deployment environments. The following instructions will enable you to authenticate your BigQuery connection in <Constant name="cloud" /> using WIF. 
 Currently, Microsoft Entra ID is the only supported identity provider (IdP). If you need additional IdP support, please contact your account team.
 
-### Set up dbt
+### 1. Set up Entra ID
+
+Create an app in Entra where dbt will request access tokens when authenticating to BigQuery via the workload identity pool:
+
+1. From the **app registrations** screen, click **New registration**.
+2. Give the app a name that makes it easily identifiable.
+3. Ensure **Supported account types** are set to “Accounts in this organizational directory only (Org name - Single Tenant).”
+4. Click **Register** to see the application’s overview screen.
+5. From the **app overview**, click **Expose an API** from the left menu.
+6. Click **Add** next to Application ID URI. The field will automatically populate. 
+7. Click **Save**.
+
+Workload Identity Federation utilizes a machine-to-machine OAuth flow that is unattended by the user; as such, a redirect URI won't need to be set for the application. Step three in this section is crucial because it determines the audience for tokens issued from the app and informs the workpool in GCP whether the calling application has permission to access the resources guarded by the workpool.
```

Comment:
```suggestion
Workload Identity Federation utilizes a machine-to-machine OAuth flow that is unattended by the user; as such, a redirect URI won't need to be set for the application. Step 3 in this section is crucial because it determines the audience for tokens issued from the app and informs the workpool in GCP whether the calling application has permission to access the resources guarded by the workpool.
```